package.json

```diff
@@ -28,6 +28,8 @@
   "dependencies": {
     "bluebird": "3.5.1",
     "body-parser": "1.18.3",
+    "chrome-launcher": "0.10.2",
+    "cli-spinner": "0.2.8",
```

Comment:
I prefer `ora` spinner by @sindresorhus :D

Reply:
good suggestion, I'll check it out and keep in mind for next time :P